src/Utilities/functions.jl

```diff
@@ -1240,6 +1240,18 @@ function operate(::typeof(/), ::Type{T}, f::MOI.ScalarAffineFunction{T},
     return operate!(/, T, copy(f), α)
 end
 
+function operate!(::typeof(/), ::Type{T}, f::MOI.ScalarQuadraticFunction{T},
+                  α::T) where T
+    f.aff_terms .= operate_term.(/, f.affine_terms, α)
+    f.quad_terms .= operate_term.(/, f.quadratic_terms, α)
+    f.constant /= α
+    return f
+end
+function operate(::typeof(/), ::Type{T}, f::MOI.ScalarQuadraticFunction{T},
```

Comment:
We can merge this method with the `ScalarAffineFunction` one by doing `f::Union{MOI.ScalarAffineFunction{T}, MOI.ScalarQuadraticFunction{T}}`